eatngo-infra/src/main/kotlin/com/eatngo/order/persistence/NotificationSsePersistenceImpl.kt

```
@@ -16,8 +18,23 @@ class NotificationSsePersistenceImpl : NotificationPersistence {
                 .apply {
                     onTimeout { emitters.remove(storeId) }
                     onCompletion { emitters.remove(storeId) }
-                }.also { sseEmitter -> sseEmitter.send(SseEmitter.event().name("CONNECTED")) }
+                    onError { emitters.remove(storeId) }
+                }.also { sseEmitter ->
+                    try {
+                        sseEmitter.send(
+                            SseEmitter.event().name(NotificationEventType.CONNECTED.eventName),
+                        )
+                    } catch (e: IOException) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)